packages/docs/src/03-Usage.svench/20-knobs.md

```diff
@@ -0,0 +1,105 @@
+<script>
+   import { View } from 'svench';
+</script>
+
+<svench:options title="Knobs" />
+
+# Knobs
+
+Knobs are used to interact with the View components.
+
+## Defining and using knobs
+
+Knobs are defined for a `View` via the `knobs` prop, and their current values
+are retrieved via `let:knobs`.
+
+```svelte
+<View name="defining and using a knob, simple" knobs={{ someProp: "my default value" }} let:knobs>
+
+    <p>someProp: {knobs.someProp}</p>
```

Comment:
Not sure I want to document this syntax... The deconstructed object is clearly the preferred way, and the other syntax can be deduced from it, if really.

My preference would be to remove this first example, and rather add "typing inference" in the deconstructor example (even better if we can limit using / insisting on the word "deconstructor"... not sure that's very user friendly).

By "typing inference", I mean that Svench will guess the kind of input to render based on the type of the default value in the shortcut (object form), for text, number, and bool. There's also a magic string syntax for ranges.

```
knobs={{
  text: "text",
  number: 123,
  checkbox: false,
  range: "-10-10;5",
}}
```

Reply:
> Not sure I want to document this syntax... The deconstructed object is clearly the preferred way, and the other syntax can be deduced from it, if really.

Fair. :-) My intent was mostly to give a slightly different example to help people grok what is coming from `knobs={...}` and what is coming from `let:knobs`, because there's a heck of a lot of knobs flying around. ^.^ 

> My preference would be to remove this first example, and rather add "typing inference" in the deconstructor example (even better if we can limit using / insisting on the word "deconstructor"... not sure that's very user friendly).

:+1: Okie.